packages/ui/src/components/w3m-box-button/index.ts

```diff
@@ -11,13 +11,14 @@ export class W3mBoxButton extends LitElement {
   // -- state & properties ------------------------------------------- //
   @property() public icon?: TemplateResult<2> = undefined
   @property() public label = ''
+  @property() public loading?: boolean = false
   @property() public onClick: () => void = () => null
 
   // -- render ------------------------------------------------------- //
   protected render() {
     return html`
-      <button @click=${this.onClick}>
-        <div>${this.icon}</div>
+      <button @click=${this.onClick} class="${this.loading ? 'disabled' : ''}">
```

Comment:
Let's use buttons disabled attribute instead, i.e. `disabled=${this.loading}` (you might need to do something weird with lit-html i.e. `.disabled=`) where that dot is used to boolean check it.